src/client/error.rs

```diff
@@ -1,11 +1,18 @@
 #[derive(Debug, thiserror::Error)]
 pub enum PlatzClientError {
-    #[error("Could not find any Platz config. Please set the PLATZ_URL and PLATZ_API_TOKEN environment variables. When running from inside a Platz deployment, mount the \"platz-creds\" secret to /var/run/secrets/platz.json .")]
+    #[error(r###"Could not find any Platz config. Use one of the following methods:
+    1. Set the PLATZ_URL and PLATZ_API_TOKEN environment variables.
+    2. Use "platz/config.toml" configuration file on your config directory.
```

Comment:
```suggestion
    2. Use "~/.config/platz/config.toml" configuration file.
```

Reply:
Right, I forgot about windows...

I just think the error should include the actual paths.

Reply:
We'll improve it later.